src/web/app/components/extension-confirm-modal/extension-confirm-modal.component.ts

```diff
@@ -1,20 +1,24 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
-import { TableComparatorService } from '../../../../services/table-comparator.service';
-import { SortBy, SortOrder } from '../../../../types/sort-properties';
-import { InstructorExtensionTableColumnModel, StudentExtensionTableColumnModel } from '../extension-table-column-model';
+import { TableComparatorService } from '../../../services/table-comparator.service';
+import { SortBy, SortOrder } from '../../../types/sort-properties';
+import {
+  StudentExtensionTableColumnModel,
+  InstructorExtensionTableColumnModel,
+} from '../../pages-instructor/instructor-session-individual-extension-page/extension-table-column-model';
 
 export enum ExtensionModalType {
   EXTEND,
   DELETE,
+  VALIDATE_DEADLINE,
```

Comment:
Not sure about this myself, just wondering if there's a better name for this - am personally at a loss as to what could be better though

Reply:
Would `SESSION-EDIT-DELETE` be slightly better?

Reply:
Would `SESSION-DELETE` be better?

Reply:
Yeah, the naming sounds better to me!